### ISSUE TITLE
[SotA Dialog] S06 Following the Shadow: provide better dialog hint about tomato surprise

### ISSUE THREAD

**Issue by max-torch** [wesnoth/wesnoth#6120]
### Text Issues: Typos, grammar mistakes, suggested improvements

* English language
* Secrets of the Ancients S06 Following the Shadow

### The Issue
We have this commentary at the top of the scenario cfg
```
# A large number of bats spawn in the cave under these two conditions:
# (1) Six non-bat units are in the cave (Bats don't stomp on the floor.)
# (2) Ardonna has been in the cave for two turns in a row. (That gets her solidly into
#     the cave, but not close to the exit. If she is alone, it's the player's fault when
#     the bats kill her.)
```
Sure the player should have to be careful and even expect tomato surprises, but I doubt that it's a strong case to expect the player to expect such a tomato surprise that involves an entire army of units instantly spawning on all sides of the leader (around the whole cave) resulting in an instant defeat. Even in other campaigns where there have been tomatoes or bats spawning, for example, The Mage in Dead Water, the surprise units still somewhat come from in front of the player.

* Please quote the text verbatim. You can use `**bold**` to point out the mistaken part:
The dialog does give caution and gives some form of warning:
           ` message= _ "If there is one bat, there are certainly more. We should be careful."`
But given the nature and degree of the tomato surprise, it justifies further hinting. Make it a clear i-told-you-so.

* _(optional)_ What do you propose to change the text to?
 ` message= _ "If there is one bat, there are certainly more. We should be careful. It would be foolish to explore this cave alone."`

### Reference Image
![image](https://user-images.githubusercontent.com/41433185/134896361-979ec408-7886-4d84-84f6-5696429e3f86.png)

**Comment by Wedge009** [comment]
This one I remember from 1.14 and I didn't find it much fun either.

**Comment by cooljeanius** [comment]
Agreed, the surprise bats got me, too, when I played this (I'm on 1.14)... I think the proposed text change would be an improvement. 

cc @beetlenaut

**Comment by max-torch** [comment]
> This one I remember from 1.14 and I didn't find it much fun either.
> 
> Edit: By the way, since you're giving feedback on the campaign design itself here and in #6119, you may be better off doing so in https://forums.wesnoth.org/viewtopic.php?t=40545.

I will make Git issues and then I will post on the forums and provide Git issue links on the posts.

**Comment by knyghtmare** [comment]
Hehe, I remember this nasty surprise. Made me game over. Did save reload and throw in 2 Necrophages and 2 Deathblades in for the second run. 

My proposal:
```
message= _ "If there is one bat, there are certainly more. We should be careful. Perhaps I should send in some ghouls inside the cave first?"
```
Or, anything that would imply to send in some expendable units to trigger the surprise so the player can plan the counter, without losing the scenario.

**Comment by max-torch** [comment]
> Hehe, I remember this nasty surprise. Made me game over. Did save reload and throw in 2 Necrophages and 2 Deathblades in for the second run.
> 
> My proposal:
> 
> ```
> message= _ "If there is one bat, there are certainly more. We should be careful. Perhaps I should send in some ghouls inside the cave first?"
> ```
> 
> Or, anything that would imply to send in some expendable units to trigger the surprise so the player can plan the counter, without losing the scenario.

It's hard to imply it this way or hint it this way since the trigger is specifically for six units to be squarely in the cave
OR
for Ardonna to be in the cave for at least 2 turns.

From the perspective of the player it's almost impossible to determine what the exact trigger is without looking at the WML.

**Comment by max-torch** [comment]
Interestingly, here's some support for the tomato hinting expressed by the maintainer and author in the forums:
![image](https://user-images.githubusercontent.com/41433185/134955028-7cfb5f58-aeee-4bce-9faf-2b0777cce75f.png)
Seems the hint didn't actually even exist initially! This proposal would be a further refinement of the hint dialog.
